examples/web/src/addPlugins.ts

```diff
@@ -4,7 +4,9 @@ import BackgroundRemovalPlugin from '@imgly/plugin-background-removal-web';
 
 async function addPlugins(cesdk: CreativeEditorSDK) {
   try {
-    cesdk.unstable_addPlugin(BackgroundRemovalPlugin());
+    cesdk.unstable_addPlugin(
+      BackgroundRemovalPlugin({ ui: { defaultLocations: 'canvasMenu' } })
```

Comment:
We prefix it with `default` cause you can change it later I guess?

Reply:
Yes, it just determines if it should be changed while loading the plugin ... by default. I thought that made it clearer. Do you think it is more confusing, @DanielHauschildt ?

Reply:
Since this is the primary place to set the location (if i understand correctly), I would be in favor of naming it `location` as well instead of `defaultLocation`